base_layer/core/tests/helpers/nodes.rs

```diff
@@ -360,7 +377,7 @@ pub fn create_network_with_3_base_nodes_with_config(
         .start(runtime, data_path);
     let (bob_node, consensus_manager) = BaseNodeBuilder::new(network)
         .with_node_identity(bob_node_identity.clone())
-        .with_peers(vec![alice_node_identity.clone(), carol_node_identity.clone()])
+        .with_peers(vec![carol_node_identity.clone()])
```

Comment:
I agree with this change, Bob will know about Alice once Alice dials him so this prevents Bob dialling Alice at the same time - same for Carol :+1: